lua/wikis/commons/Widget/EmptyPagePreview/Team.lua

```diff
@@ -0,0 +1,454 @@
+local Widget = Lua.import('Module:Widget')
+local WidgetUtil = Lua.import('Module:Widget/Util')
+
+local Condition = Lua.import('Module:Condition')
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+local ConditionUtil = Condition.Util
+
+---@class EmptyTeamPagePreview: Widget
+---@operator call(table): EmptyTeamPagePreview
+local EmptyTeamPagePreview = Class.new(Widget)
+
+---@return Widget?
+function EmptyTeamPagePreview:render()
+	if not Namespace.isMain() then
+		return
+	end
+
+	self.team = TeamTemplate.getPageName(self.props.pageName or mw.title.getCurrentTitle().prefixedText)
+
+	if not self.team then return end
+
+	self.teams = TeamTemplate.queryHistoricalNames(self.team)
+
+	local rosterFromLastPlacement = Logic.readBool(self.props.rosterFromLastPlacement)
+
+	return HtmlWidgets.Div{
+		children = WidgetUtil.collect(
+			HtmlWidgets.H2{children = {'Overview'}},
+			self:_infobox(),
+			rosterFromLastPlacement and self:_rosterFromLastPlacement() or self:_rosterFromTransfers(),
+			self:_matches(),
+			self:_results(),
+			HtmlWidgets.H2{children = {'References'}}
+		),
+	}
+end
+
+---@private
+---@return Html
+function EmptyTeamPagePreview:_infobox()
+	local data = self:_getNationalitiesAndCoachesFromLastPlacement()
+
+	local coaches
+	if Logic.isNotEmpty(data.coaches) then
+		coaches = HtmlWidgets.Fragment{
+			children = Array.interleave(Array.map(data.coaches, function(coach)
+				return HtmlWidgets.Fragment{
+					children = {
+						Flags.Icon{flag = coach.flag},
+						'&nbsp;',
+						Link{link = coach.pageName, children = {coach.displayName}},
+					}
+				}
+			end), HtmlWidgets.Br{})
+		}
+	end
+
+	local regionCounts = {}
+	local location
+	local rosterRegion
+	local rosterSize = 0
+	for _, countryRosterSize in pairs(data.nationalities or {}) do
+		rosterSize = rosterSize + countryRosterSize
+	end
+	for country, countryCount in pairs(data.nationalities or {}) do
+		local region = Region.name{country = country}
+		if countryCount > (rosterSize / 2) then
+			location = country
+			rosterRegion = region
+			break
+		end
+		if region then
+			regionCounts[region] = (regionCounts[region] or 0) + 1
+			if regionCounts[region] > (rosterSize / 2) then
+				location = region
+				rosterRegion = region
+				break
+			end
+		end
+	end
+	location = location or 'World'
+
+	local games = self:_fetchGamesFromPlacements()
+
+	local args = {
+		location = location or 'World',
+		queryEarningsHistorical = Logic.nilOr(Logic.readBoolOrNil(self.props.queryEarningsHistorical), true),
+		doNotIncludePlayerEarnings = Logic.nilOr(Logic.readBoolOrNil(self.props.doNotIncludePlayerEarnings), true),
+		name = TeamTemplate.getRaw(self.team).name,
+		coaches = coaches,
+		region = self:_determineRegionFromPlacements() or rosterRegion,
+		wiki = EmptyTeamPagePreview._getWiki(self.props, games),
```

Comment:
Is the wiki parameter still in use? Wasn't it removed?

Reply:
generally speaking yes it is still supported and needed/used in infoboxes on some wikis
(e.g. lotv, hots, wol for sc2 with infoboxes league, series, unit, building and some more)

but we likely can ignore it for teams and players

Reply:
I'd consider that legacy behavior :P
Thus i'd propose to not add it in here.
```suggestion
```